### ISSUE TITLE
プロダクトバックログ

### ISSUE THREAD

**Issue by cloudliner** [terra-yucco/ruthenium#3]

**Comment by cloudliner** [comment]
9/29 20:00
![img_0134](https://cloud.githubusercontent.com/assets/594586/18952497/ce04cfec-8684-11e6-9c27-7cfcbcc5f992.JPG)

**Comment by b1621ry** [comment]
すいません、付箋を見てて思ったのですが、家にある調理器具の種類によっても料理できるものとできないものがあるような気がしたのですが、、、

**Comment by cloudliner** [comment]
**9/29 20:40**
![img_0136](https://cloud.githubusercontent.com/assets/594586/18952770/29f7817c-8686-11e6-83b5-cee4f481b261.JPG)

**Comment by cloudliner** [comment]
確かに、調理器具の種類によっても出来るものは違いますよね。あとは、必要な調味料が常備しているものでないと「その料理のためだけに調味料を買うのか？」とかあります。

**Comment by b1621ry** [comment]
いちいち調味料や調理器具の有無を全て登録させるのは煩雑で、使いたくなくなりそうで、作ったことがある料理を登録してもらえば、そこからその料理を作るために必要な調味料や調理器具の有無をある程度推察できるような気がします。

**Comment by cloudliner** [comment]
作れない or 作りたくないレシピが出てきたら「チェンジ」する機能があるといいと思います。
あとは、理想的にはチェンジされた「レシピ」に共通の要素を学習して、、、

**Comment by cloudliner** [comment]
**9/29 21:30**
![img_0138](https://cloud.githubusercontent.com/assets/594586/18954788/75600342-868f-11e6-8913-66914bf33752.JPG)

**Comment by terra-yucco** [comment]
>個人的には、作れない or 作りたくないレシピが出てきたら「チェンジ」する機能があるといいな、と思います。

あ、これはいいね！

**Comment by cloudliner** [comment]
**10/01 18:00**
![2016-10-01 17 56 57](https://cloud.githubusercontent.com/assets/594586/19014040/61c567f2-881d-11e6-99a7-b75ddab4bc40.jpg)
